.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: required # Required for apt-get build-dep
+dist: trusty
+language: cpp
+compiler: gcc
+
+env:
+  - MUMBLE_QT=qt5 MUMBLE_HOST=x86_64-linux-gnu
```

Comment:
Nvm, I see the before install script uses these envs to install stuff.

Why does it not work then?